### DISCUSSION TITLE
[Bug] Misconfigured Recaptcha on multiversx.com Blog

### DISCUSSION THREAD

**Discussion by michavie** [ElrondNetwork/community#17] [General]
The Google Recaptcha seems misconfigured on the email signup form on https://multiversx.com/blog, preventing one from signing up.

Posted in the 'General' category – an idea would be to create a 'Bugs' category for issues like these that don't refer to any particular public repository.

![Screenshot 2022-12-23 at 4 27 27 PM](https://user-images.githubusercontent.com/39144548/209360093-400fe1a4-de48-4ee1-aaaf-79aa2cc940ab.png)

**Comment by schimih** [comment]
Thanks for your message @michavie . We will look into it. :raised_hands:

**Comment by schimih** [comment]
Can you please try it again? Does the issue persist?

**Reply by michavie** [reply]
Captcha works now, and verification mail arrives too, opening the link shows an error page:

![Screenshot 2023-01-10 at 4 52 56 PM](https://user-images.githubusercontent.com/39144548/211598652-edb997e7-8398-468b-93a7-4ca43b484c30.png)

![Screenshot 2023-01-10 at 4 53 08 PM](https://user-images.githubusercontent.com/39144548/211598720-b59f5739-9fbf-4f7f-bace-c36a66602039.png)
